custodia/secrets.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, config, section):
 
     def _db_key(self, trail):
         if len(trail) < 2:
+            self.logger.error('Forbidden action. Trail: %s has less than two '
```

Comment:
Trail is an internal concept, if we log at error level we need to provide a message that is comprehensible to the admin looking at the logs, w/.o needing him to know how we internally manage to break stuff a apart.

In this specific case I would say the message should say something about the fact this operation can only done within a container or some similar message. We cannot say "not top-level" because we can have prefixes in paths.